### PR TITLE
Edit legal docs 

### PR DESCRIPTION
**Pull request by benmccallum**
Modifications I've made to the original docs generated from getterms.io

### CODE REVIEW

**Review by PascalSenn — COMMENTED** [review]
Just read through it, looks good 👍 
Couple of questions:
1. In some places it's referred to 'website and/or services' in other places only 'services'
2. Is it 'Services' or 'services'

File: website/src/pages/legal/terms-of-service.html [hunk 1]
```diff
@@ -100,35 +130,62 @@ <h3>Liability</h3>
...
+
 <h3>Links</h3>
 <p>
-  ChilliCream Inc. has not reviewed all of the sites linked to its website and
-  is not responsible for the contents of any such linked site. The inclusion of
-  any link does not imply endorsement, approval, or control by ChilliCream Inc.
-  of the site. Use of any such linked site is at your own risk and we strongly
-  advise you make your own investigations with respect to the suitability of
-  those sites.
+  ChilliCream Inc. has not reviewed all of the sites linked to by our Services
```
[the remark this answers is not in the document]
Comment (benmccallum) [reply website/src/pages/legal/terms-of-service.html:167]: I think this one is ok, our we could make it "from our Services"

**Comment by benmccallum** [comment]
> In some places it's referred to 'website and/or services' in other places only 'services'

Yea we could probably normalize to just "Services".

> Is it 'Services' or 'services'

Should be capital given other examples I've seen.

File: website/src/pages/legal/acceptable-use-policy.html [hunk 2]
```diff
@@ -1,4 +1,7 @@
 <h2>Acceptable Use Policy</h2>
+<p>This policy is effective as of 26 August 2021.</p>
+<p>Last updated: 26 August 2021</p>
+
 <p>
   This acceptable use policy covers the products, services, and technologies
   (collectively referred to as the &ldquo;Products&rdquo;) provided by
```
[the remark this answers is not in the document]
Comment (benmccallum) [reply website/src/pages/legal/acceptable-use-policy.html:7]: Changed everything to Services to align with other documents.

**Pull request merged**